CMakeLists.txt

```diff
@@ -840,7 +755,6 @@ foreach(__CLDNN_CompilerFlagName IN ITEMS "CMAKE_CXX_FLAGS" "CMAKE_C_FLAGS")
       ALL_PATTERN ""
       SET
         CompileAsDefault
-        DebugInfoProgramDatabase
```

Comment:
[MINOR] If you want to disable all debug info you should use **DebugInfoDisabled**. Otherwise it uses debug settings already contained in `CMAKE_C_FLAGS` and `CMAKE_CXX_FLAGS`.

If you use **DebugInfoDisabled** and you want debug information/symbols on debug add **DebugInfoProgramDatabase** to intel_config_flag_apply_settings() from lines 775-782 (with PATTERN "^Debug").